app/routes.py

```diff
@@ -1,2 +1,233 @@
-from flask import Blueprint
+import datetime, requests, os
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import Blueprint, jsonify, make_response, request
+from dotenv import load_dotenv
 
+load_dotenv()
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+# creates a Task
+@tasks_bp.route("", methods=["POST"], strict_slashes=False)
+def create_task():
+    request_body = request.get_json()
+
+    if "title" not in request_body or "description" not in request_body or "completed_at" not in request_body:
+        return make_response(
+            {"details": f"Invalid data"}, 400
+        )
+
+    new_task = Task(
+        title = request_body["title"],
+        description = request_body["description"],
+        completed_at = request_body["completed_at"]
+    )
+
+    db.session.add(new_task)
+    db.session.commit()
+
+    return make_response(
+        {"task":(new_task.to_dict())}, 201
+    )
+    
+# reads all created Tasks
+@tasks_bp.route("", methods=["GET"], strict_slashes=False)
+def get_tasks():
+    tasks_response = []
+
+    if request.args.get("sort") == "asc":
+        tasks = Task.query.order_by(Task.title)
+    elif request.args.get("sort") == "desc":
+        tasks = Task.query.order_by(Task.title.desc())
+    else:
+        tasks = Task.query.all()
+
+    for task in tasks:
+        tasks_response.append(task.to_dict())
```

Comment:
This works well and is very readable, but if you'd like to incorporate list comprehensions into your code, a loop that builds a list like this is a great place to start:
```suggestion
    tasks_response = [task.to_dict() for task in tasks]
```